src/shared/utils/asset.ts

```diff
@@ -12,10 +12,18 @@ import {
   AssetRuneERC20,
   AssetRuneERC20Testnet
 } from '@xchainjs/xchain-thorchain'
+import { assetFromStringEx } from '@xchainjs/xchain-util'
+
+const AssetSynthBtc = assetFromStringEx('BTC/BTC')
```

Comment:
Is there a reason why other synth assets are not added? E,g, ETH/ETH?
I see in render/helpers/assetHelpers isBtcAssetSynth is added for synth asset comparison but there is nothing for ETH and other L1 assets.

Reply:
at this stage I don't think they are needed that much